packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -38,6 +39,7 @@ interface AuthConfig extends ApiClientConfigParams {
     silenceWarnings?: boolean
     logger: Logger
     defaultDnt?: boolean
+    refreshTokenTTL?: number
```

Comment:
nit, would we consider renaming this to `refreshTokenCookieTTL` or something else that explicitly mention "cookie". This is to make sure that developers don't have the illusion of changing the actual token TTL. It's important to know that the token is valid for 90 days regardless of this option, this option only controls the cookie.

Reply:
done ✅